ansible/roles/csm.ims-remote/README.md

```diff
@@ -0,0 +1,57 @@
+csm.ims-remote
+==================
+
+Configure a compute image to be able to build IMS images
+
+Requirements
+------------
+
+The system is able to access the `ims-remote-third-party` repo in Nexus.
+
+Role Variables
+--------------
+
+Available variables are listed below, along with default values (located in
+`defaults/main.yml`):
+
+```yaml
+    builder_architecture: x86_64
+```
+
+The architecture of the compute node to be customized. 
```

Comment:
Isn't this something that can be detected from the image being customized? Or does this not need to match? Can you set this to x86_64 even if the image being customized is an Arm image? I figured this was being used to install RPMs and thus should match the architecture of the image. 

I assume the architecture of the target is something that you can get from an Ansible inventory, but @rbak-hpe or @jsl-hpe would know for sure.

Reply:
I discussed this with @dlaine-hpe . He thinks that we do want the architecture here to match the image architecture. So I'll ping @rbak-hpe and @jsl-hpe to see if this is indeed something that the play can query directly from Ansible, rather than needing to get it from a variable. (If it's in a variable, then in order to use this for Arm, customers will need to add commits to VCS to set that variable, which is fine if it's necessary, but in this case it sounds like it isn't)

Reply:
Also worth noting that even if this is a variable there's no need to commit to VCS to change it.  CFS supports an "ansible-passthrough" that you can use to pass in "extra-vars" and set this value when you create the session.